Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/OnDemand/Utils/Common.hs

```diff
@@ -1786,6 +1786,12 @@ getRiderName req = do
       tagValue = Utils.getTagV2 Tags.RATING_TAGS Tags.RIDER_NAME tagGroups
    in tagValue
 
+getCancellationReason :: Spec.CancelReq -> Maybe Text
+getCancellationReason req = do
+  let descriptor = req.cancelReqMessage.cancelReqMessageDescriptor
+      reason = descriptor >>= (.descriptorShortDesc)
+   in reason
```

Comment:
no need of in as you are using do operator at top.

Reply:
req.cancelReqMessage.cancelReqMessageDescriptor >>= (.descriptorShortDesc)
you can do it in one line

Reply:
Done